src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -5,20 +5,19 @@ authors:
   - mfriesenhahn
 date: 2019-06-26
 description: |
-  This handbook helps contributors to web.dev create effective, engaging content
-  and get it published as easily as possible.
+  Learn how to write for an international audience.
 ---
 
-web.dev has a large international audience. Members of that audience should be able to access and understand web.dev content, and they should feel included in the ways we speak to our readers.
+web.dev has a large international audience. We want members of that audience to access and understand web.dev content, and they should feel included in the ways we speak to our readers.
 
 ## Use readable language
-Aim for a grade 8 reading level when possible. If you don't have a feel for what grade 8 texts look like, check out some [examples on Newsela](https://newsela.com/articles/#/rule/latest?grade_levels=8.0). If you're feeling frisky, you can also run your text through a readability test.
```

Comment:
ah! frisky??

Reply:
What's wrong with _frisky_? Anyway, changed to _ambitious_ and kept this guidance. I do want non-experts to have a way to calibrate on what more accessible writing looks like.

Reply:
I've stated elsewhere my issues with reading levels. This has another problem: "grade 8 reading level". 16 of the 29 people in Chrome Developer Relations are not products of the American education system.